src/services/accountService.js

```diff
@@ -21,14 +22,25 @@ export const completeRegistration = async ({ logo, bio, contacts }) => {
         const json = await res.json();
 
         if (!res.ok) {
+
+            createEvent(EVENT_TYPES.ERROR(
+                "Complete Registration",
```

Comment:
Make this a constant at the top of the file, so that all events here are consistent.

(This goes for all service metrics in other files as well)